### PR TITLE
Fix the bug that getnextblockvalidators return empty when there is no votes.

### PR DESCRIPTION
**Pull request by chenzhitong**
close https://github.com/neo-project/neo-modules/issues/433

### CODE REVIEW

File: src/RpcServer/RpcServer.Blockchain.cs [hunk 1]
```diff
@@ -197,14 +197,29 @@ protected virtual JObject GetNextBlockValidators(JArray _params)
...
+                    return validator;
+                }).ToArray();
+            }
+            else
+            {
+                return validators.Select(p =>
+                {
+                    JObject validator = new JObject();
+                    validator["publickey"] = p.ToString();
+                    validator["votes"] = 0;
```
Comment (chenzhitong) [inline src/RpcServer/RpcServer.Blockchain.cs:218]: 0 or 100000000 whis is better?

Comment (shargon) [reply src/RpcServer/RpcServer.Blockchain.cs:218]: 0 it's fine for me

**Review by shargon — APPROVED** [review]

**Review by vncoelho — COMMENTED** [review]
I think it should return the Default Commit instead.
Otherwise it will be strange to have "0" and, in fact, we actually have the next validators.

**Review by vncoelho — COMMENTED** [review]
Sorry, I think it is good like this, perhaps the "active" will contain the current validators.

File: src/RpcServer/RpcServer.Blockchain.cs (hunk 1 above)
Comment (cloud8little) [reply src/RpcServer/RpcServer.Blockchain.cs:218]: I think this API should be sync with the result that neo-cli client returns. getnextblockvalidators should return nextblockvalidators, and we can have GetCandidates to return the candidates information.

**Review by Tommo-L — APPROVED** [review]

**Pull request merged**